contracts/pizzabonus/tables.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <eosio/eosio.hpp>
+
+struct [[eosio::table]] coupon {
+  uint16_t value;
+  bool active = false;
+};
+
+// EOSIO_REFLECT(Coupon, value, active);
```

Comment:
This macro did not work, it is pending to investigate if this is part of the new version